LoopKitUI/ServiceUI.swift

```diff
@@ -8,28 +8,39 @@
 
 import LoopKit
 import SwiftUI
+import HealthKit
+
+public protocol SupportInfoProvider {
```

Comment:
Also, not sure this belongs in "service". Perhaps we should consider breaking out these "service" concepts and keeping them all separate. Later, though.

Reply:
+1

Reply:
Yeah, I could see some refactoring of these responsibilities out Type-wise; they do have strong relationships with the services (settings being stored in tidepool/nightscout backend, support being handled by tidepool backend, etc.) that would still exist, but I'm sure there are improvements that could be made to the type system. I do think this should be done as a later refactoring, though, if we think that functionally this is not problematic.